extensions/objects/templates/literally-canvas-diagram-editor.component.ts

```diff
@@ -16,14 +16,35 @@
  * @fileoverview Directive for literally canvas diagram editor.
  */
 
+require('domain/utilities/url-interpolation.service.ts');
+require('pages/exploration-player-page/services/image-preloader.service.ts');
+require('services/alerts.service.ts');
+require('services/assets-backend-api.service.ts');
+require('services/context.service.ts');
+require('services/csrf-token.service.ts');
+require('services/image-local-storage.service.ts');
+require('services/image-upload-helper.service.ts');
+
 const LC = require('literallycanvas');
 require('literallycanvas/lib/css/literallycanvas.css');
 require('services/literally-canvas-helper.service.ts');
 
 angular.module('oppia').component('literallyCanvasDiagramEditor', {
   template: require('./literally-canvas-diagram-editor.component.html'),
+  bindings: {
+    value: '='
```

Comment:
Perhaps use a clearer name.

Reply:
This is the convention that is used across all rte components.